OrbitGl/PresetsDataView.cpp

```diff
@@ -55,17 +101,20 @@ std::string PresetsDataView::GetToolTip(int row, int /*column*/) {
     return OrbitUtils::Compare(presets_[a]->Member, presets_[b]->Member, ascending); \
   }
 
+#define ORBIT_PRESET_FUNC_SORT(Func) \
+  [&](int a, int b) { return OrbitUtils::Compare(Func(presets_[a]), Func(presets_[b]), ascending); }
+
```

Comment:
We should avoid adding new macros to the code base. I see no disadvantage in adding this code directly to `DoSort`.

Reply:
EDIT: If there is a reason for having a macro that I don't see, we should at least `#undef` the macro after usage.